discover/deployment/articles/06-upgrading-to-liferay-7-1/03-running-the-upgrade-process/01-running-the-upgrade-tool.markdown

```diff
@@ -213,8 +213,11 @@ Specify the following information to configure the upgrade itself:
 
 **liferay.home:** the [Liferay home folder](/discover/deployment/-/knowledge_base/7-1/installing-liferay#liferay-home) *(required)*
 
+**dl.store.impl:** the document library server will use this implementation to persist documents. Copy the same value you are using in your current portal-ext.properties (required when you do not use the default implementation `com.liferay.portal.store.file.system.FileSystemStore`)
+
```

Comment:
@achaparro In [preparing for the upgrade](https://github.com/liferay/liferay-docs/blob/7.1.x/discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1/00-preparing-for-upgrade-to-liferay-7-1-intro.markdown#step-5-update-your-portal-properties-), they update their portal properties and doc lib store. Won't the `dl.store.impl` setting from their portal properties be sufficient for the upgrade tool to use? 

thanks

Reply:
You are right, we point to that article in the running the upgrade article however it seems a bit confused for customers since three of them (for all I know) forgot to add this property and that causes that the upgrade fails and they have to repeat it.

We should clarify this point somehow, reading the article you have pointed out seems that we don't need this property anymore:

> _and you must configure the appropriate updated implementation name rather than copying over the name from your 6.2 portal-ext.properties._

But the property dl.store.impl is still in portal.properties, do you have any idea to clarify it?

Thanks.

Reply:
I see your point. I'll rewrite that to clarify that you must still specify your `dl.store.impl`, but update the impl class' fully qualified name.

I'll send you a change for it. Thanks!